src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -3764,9 +3764,7 @@ paths:
       tags:
       - Cluster
     post:
-      description: "\n        Add a ceph user with its defined capabilities.\n   \
-        \     :param user_entity: Entity to change\n        :param capabilities: List\
-        \ of capabilities to add to user_entity\n        "
+      description: "Add a Ceph user, with its defined capabilities."
```

Comment:
@zdover23 the proper way of updating the RESTAPI doc should be done through its respective controller. There is a documentation which describes the [process of doing it](https://docs.ceph.com/en/latest/dev/developer_guide/dash-devel/#rest-api-documentation). Its done using a tox script, which helps to generate the documentation directly from the code rather than updating it manually. So the place where this needs to be updated is in the [controllers/ceph_users.py](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/controllers/ceph_users.py#L51), and then regenerate the openapi doc using `tox -e openapi-fix` from the `mgr/dashboard` folder.

This should be actually fixed by updating the `@EndpointDoc` decorator rather than updating the comment which might parse the documentation properly.

https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/controllers/ceph_users.py#L195
This line should be updated something like below.
```
create=CRUDCollectionMethod(
        func=CephUserEndpoints.user_create,
        doc=EndpointDoc("Add a ceph user, with its defined capabilities."
                                          parameters={
                                                     "user_entity": Param(str, "Entity to add")
                                                     "capabilities": Param([{"entity": str}, "cap": "allow *"}], "Capabilities")
                                                     }
                                               )
    ),
```
I see there is a tracker, if you want you can assign it to me and probably I can take care of it. I see there are couple of issues with the documentation in the file. But the automation is good for us so that we don't unintentionally break the documentation in a way that it doesn't match with what we have in the code.

Reply:
@nizamial09 That's cool with me. You can take it. 

I'd like to learn how to update this stuff sometime, though, in case of documentation emergency.

Reply:
Sounds good @zdover23. When I raise a PR I'll also tag you there and probably from it you can understand the code changes. And later on you can look into the tox script to generate the documentation locally as well.